web/js/layers/wv.layers.modal.js

```diff
@@ -689,6 +690,7 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
    */
   var drawAllLayers = function() {
     var projection = models.proj.selected.id;
+
```

Comment:
Let's get rid of this whitespace insertion, and squash this PR into a single commit since it's a single change.